pkg/deploy/elbv2/listener_rule_manager.go

```diff
@@ -47,39 +53,46 @@ func (m *defaultListenerRuleManager) Create(ctx context.Context, resLR *elbv2mod
 	if err != nil {
 		return elbv2model.ListenerRuleStatus{}, err
 	}
+	ruleTags := m.trackingProvider.ResourceTags(resLR.Stack(), resLR, resLR.Spec.Tags)
+	req.Tags = convertTagsToSDKTags(ruleTags)
 
 	m.logger.Info("creating listener rule",
 		"stackID", resLR.Stack().StackID(),
 		"resourceID", resLR.ID())
-	var sdkLR *elbv2sdk.Rule
+	var sdkLR ListenerRuleWithTags
 	if err := runtime.RetryImmediateOnError(m.waitLSExistencePollInterval, m.waitLSExistenceTimeout, isListenerNotFoundError, func() error {
 		resp, err := m.elbv2Client.CreateRuleWithContext(ctx, req)
 		if err != nil {
 			return err
 		}
-		sdkLR = resp.Rules[0]
+		sdkLR = ListenerRuleWithTags{
+			ListenerRule: resp.Rules[0],
```

Comment:
the tags should be set here to be  ruleTags  to be semantically consistent.

Reply:
It wasn't used, so I didn't set it earlier. Added the tags in the new diff.